_includes/footer.html

```diff
@@ -8,4 +8,9 @@
   <div class="sm:w-1/2">
     {% include creativeCommons.html %}
   </div>
-</div>
\ No newline at end of file
+</div>
+<div class="flex flex-col sm:flex-row justify-between items-baseline gap-4">
```

Comment:
I don't think most of the tailwind styles are needed here, as it's not containing flex items. You could also specify `"my-4"` instead of beginning the content with a `<br>`

Reply:
Updated - yes, that's a lot cleaner.